messages_extends/tests.py

```diff
@@ -123,4 +123,10 @@ def _patched_queryset(*args, **kw):
             no_called.append(1)
         ps._message_queryset = _patched_queryset
         ps._get()
-        self.assertEquals(no_called[0], 1)
\ No newline at end of file
+        self.assertEquals(no_called[0], 1)
+
+    def test_delete(self):
+        messages.add_message(self.client, WARNING_PERSISTENT, "Warning Test")
+        self.assertEquals(Messages.objects.count(), 1)
+        messages.delete()
```

Comment:
Django changed functionality in 2.2 such that models that override `__eq__` also need to override `__hash__`: https://code.djangoproject.com/ticket/30333

```suggestion
        user = self._get_user()
        self.client.login(username=user.username, password='password')
        messages.add_message(self.client, WARNING_PERSISTENT, "Warning Test")
        self.assertEquals(Messages.objects.count(), 1)
        user.delete()
```